v0/destinations/ga4/transform.js

```diff
@@ -34,6 +34,33 @@ const {
   GA4_PARAMETERS_EXCLUSION
 } = require("./utils");
 
+/**
+ * Returns GA4 client_id
+ * @param {*} message
+ * @param {*} Config
+ * @returns
+ */
+const getGA4ClientId = (message, Config) => {
+  // first we will search from webapp
+  if (Config.clientIdFieldIdentifier) {
+    const { clientIdFieldIdentifier } = Config;
```

Comment:
we can do object destructuring in the function parameter itself.

Reply:
updated